packages/frontend/src/pages/public/ManageAccountPage.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Avatar,
+  Button,
+  Col,
+  Container,
+  Input,
+  Link,
+  Modal,
+  Row,
+  Text,
+} from '@nextui-org/react';
+import { getAuth } from 'firebase/auth';
+import React from 'react';
+import { useAuth } from '../../hooks/useAuth';
+
+interface ManageAccountPageProps {}
+
+const btnStyle = {
+  width: '100%',
+  padding: '30px',
+};
+
+const ManageAccountPage: React.FC<ManageAccountPageProps> = (
+  props: ManageAccountPageProps,
+) => {
+  const [visible, setVisible] = React.useState(false);
+  const handler = () => setVisible(true);
+  const closeHandler = () => {
+    setVisible(false);
+  };
+
+  function handleJoiningHouse(): void {
+    // pass the data to the backend
+    // direct the page to the house dahsboard
+  }
+
+  // get the house status, if its new users showing the account management page
+  // if not, direct to the house they joined
+  const userData = useAuth().user;
+
+  return (
+    <div className="h-screen overflow-y-auto bg-gradient-to-b from-land_page_bg_start to-land_page_bg_end">
+      <Avatar
+        text={userData.displayName}
```

Comment:
```suggestion
        text={user?.displayName}
```

Reply:
It cause the type error. <img width="385" alt="image" src="https://user-images.githubusercontent.com/49899381/157575218-7c69cb38-4f7f-43ec-8331-4e9dbc7647a2.png">

Reply:
`user?.displayName ?? ""`

Reply:
Alternatively, you can get the users profile picture. Something like
```suggestion
       icon={<img alt="Avatar" className="rounded-full" src={user?.photoURL ?? ''} />}
```
may be more appropriate.

Reply:
Understand, so we can get the profile image from the google authentication. I will update it.